src/onnx/onnx.cpp

```diff
@@ -2486,6 +2445,28 @@ struct onnx_parser
         return l;
     }
 
+    instruction_ref
```

Comment:
After looking back at `parse_equal`, `parse_greater`, and `parse_less`, they are all basically the same except the operator name. You could probably refactor these functions into using just one `parse_compare` function, which takes in an additional parameter of `op_name` (see `parse_reduce_oper` function for an example).

You will also need to change L139, L145, and L150 to pass in the second argument of the `op_name` (see L164-168 as an example).